src/pages/MyPatches.tsx

```diff
@@ -1,5 +1,27 @@
 import React from "react";
+import { useLocation, useHistory } from "react-router-dom";
+import queryString from "query-string";
+import Checkbox from "@leafygreen-ui/checkbox";
 
 export const MyPatches = () => {
-  return <div>My Patches Page</div>;
+  const { search, pathname } = useLocation();
+  const history = useHistory();
+  const parsed = queryString.parse(search);
+  let showCommitQueue = (parsed.commitQueue === "true");
+  return (
+    <div>
+      <div>My Patches Page</div>
+      <Checkbox
+        onChange={() => {
```

Comment:
I'm not sure if this should be in its own separate function, left it inline for now

Reply:
moved to a function after discovering the linter takes issue with lambdas